src/components/simulator/SimulatorStrategyType.tsx

```diff
@@ -0,0 +1,105 @@
+import { FC, useRef, KeyboardEvent } from 'react';
+import { Tooltip } from 'components/common/tooltip/Tooltip';
+import { ReactComponent as IconTwoRanges } from 'assets/icons/two-ranges.svg';
+import { ReactComponent as IconOverlappingStrategy } from 'assets/icons/overlapping-strategy.svg';
+import { cn } from 'utils/helpers';
+import { SimulatorType } from 'libs/routing/routes/sim';
+import { useNavigate } from '@tanstack/react-router';
+
+interface Props {
+  strategyType: SimulatorType;
+}
+
+export const SimulatorStrategyType: FC<Props> = ({ strategyType }) => {
+  const list = useRef<HTMLUListElement>(null);
+  const navigate = useNavigate();
+
+  const items: {
```

Comment:
lets declare a local interface for readability

Reply:
done